code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -471,10 +472,21 @@
 
 /datum/chemical_reaction/reagent_explosion/nitrous_oxide
 	required_reagents = list(/datum/reagent/nitrous_oxide = 1)
-	strengthdiv = 7
+	strengthdiv = 9
 	required_temp = 575
 	modifier = 1
 
+/datum/chemical_reaction/reagent_explosion/nitrous_oxide/on_reaction(datum/reagents/holder, created_volume)
+	holder.remove_reagent(/datum/reagent/sorium, created_volume*2)
+	var/turf/turfie_owo = get_turf(holder.my_atom)
```

Comment:
>=|

Reply:
perfectly normal var names, they are both descriptive and provide an inaight into the coders state of mind.

Reply:
this is based as hell

Reply:
needs to be addressed

Reply:
do as i say not as i do

Reply:
sure bro